www/src/data/sidebars/doc-links.yaml

```diff
@@ -154,9 +154,6 @@
           link: /docs/source-plugin-tutorial/
     - title: Starters
       link: /docs/starters/
-      items:
```

Comment:
Wanted to surface that there are use cases for starters that don't involve sharing them (eg, agencies building on Gatsby often have an internal starter they use for all projects) -- so we may want to keep this link in.

Reply:
I considered leaving it there, but it's a pretty messy IA to link into the Contributing section from the middle of the docs sidebar. I'd recommend adding more language to the Starters doc instead. Especially since this has been a stub until now...

Reply:
Ok so after looking at the Plugins section of the docs, I decided to keep this in `/docs` instead of `/contributing`. I also reworded the title to match the style of other docs ("Creating a Starter" instead of "Create a Starter"). Thanks for the suggestion!